### ISSUE TITLE
The 'portage<4,>=3' distribution was not found and is required by lostfiles

### ISSUE THREAD

**Issue by xsoalokinx** [gcarq/portage-lostfiles#6]
Running this on a gentoo system produces the following error.

Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.11/lostfiles", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/lib/python3.11/site-packages/lostfiles.py", line 316, in main
    args = parse_args()
           ^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/lostfiles.py", line 310, in parse_args
    version="%(prog)s {}".format(pkg_resources.require("lostfiles")[0].version),
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pkg_resources/__init__.py", line 968, in require
    needed = self.resolve(parse_requirements(requirements))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pkg_resources/__init__.py", line 829, in resolve
    dist = self._resolve_dist(
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pkg_resources/__init__.py", line 870, in _resolve_dist
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'portage<4,>=3' distribution was not found and is required by lostfiles

**Comment by gcarq** [comment]
I can reproduce this locally. However I'm not yet sure why portage is no longer listed with `pip list`, although it still lives within `/usr/lib/python3.11/site-packages/`:

```
$ equery f portage
...
/usr/lib/python3.11/site-packages/portage
/usr/lib/python3.11/site-packages/portage/__init__.py
...
```

**Comment by gcarq** [comment]
Should be fixed with the latest version (v0.5.4) available in guru. Feel free to reopen if it still persists
